### PR TITLE
Translation tr

### LINKED ISSUE #50: Translation - TR
Translation of the pages content to Turkish
Check the typos and make sure that everything is coming from i18next

### PR DESCRIPTION
**Pull request by knargocer**
- Added the translation for Turkish
- Made the WhoWeAre component use i18next
- Changed the link in the post component 

close #50

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ i18n
   // for all options read: https://www.i18next.com/overview/configuration-options
 
   .init({
-    fallbackLng: "en",
+    fallbackLng: "tr",
```
Comment (Ammar-64) [inline src/i18n.js]: I think this is the default language it should remain en

**Review by Ammar-64 — COMMENTED** [review]
Can you take a look at the comment I added

**Comment by knargocer** [comment]
@Ammar-64  PTAL

File: src/locales/tr.json [hunk 2]
```diff
@@ -1,8 +1,179 @@
...
+      "failUpdate": "Profil yenilenemedi",
+      "updateProfile": "Profilini yenile",
+      "email": "Email",
+      "password": "Şifre",
+      "passConfirm": "Şifre Doğrulama",
+      "update": "Güncelle",
+      "cancel": "İptal"
+    }
+  },
+  "about": {
```
Comment (Ammar-64) [inline src/locales/tr.json]: I think there are some updated stuff here if you look at the English file you’ll see them

File: src/locales/tr.json [hunk 3]
```diff
@@ -1,8 +1,183 @@
...
+      {
+        "title": "Sitenin Anafikri",
+        "paragraph": "Bağış yapmaya istekli kişiler ile bunlara ihtiyacı olan kişiler arasındaki bağlantıyı sağlamak ve bağışı en çok ihtiyacı olan kişilere ulaştırmak."
+      },
+      {
+        "title": "Hakkımızda",
+        "paragraph": "Bir grup genç insan, eşitsizliğim olmadığı daha iyi bir dünyanın hayalini kuruyor ve ihtiyacı olan insanlara ihtiyaç duyulan desteği sağlama çabalarını entegre etmeye çalışıyor. Kendimizi geliştirmek için her zaman yeni fikirler edinmeye çalışıyoruz."
+      }
+    ],
+    "team": [
```
Comment (Ammar-64) [inline src/locales/tr.json]: I meant the whole about object even this team one, it’ll be better if you open the English beside the turkish one and compare every object in both files

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Comment by Ammar-64** [comment]
@betulerkoc, @orcunsagirsoy and @Luey-A if one of you who know a clue about turkish can review this, I’ll appreciate it, cause I accepted only the code and don’t know what’s is really happening 😅

**Review by betulerkoc — APPROVED** [review]

**Review by olaouso — APPROVED** [review]

**Pull request merged**